config/src/test/scala/com/typesafe/config/impl/ConfigBeanFactoryTest.scala

```diff
@@ -46,10 +46,9 @@ class ConfigBeanFactoryTest extends TestUtils {
             ConfigBeanFactory.create(config, classOf[ValidationBeanConfig])
         }
 
-        val expecteds = Seq(Missing("propNotListedInConfig", 77, "string"),
-            WrongType("shouldBeInt", 78, "number", "boolean"),
-            WrongType("should-be-boolean", 79, "boolean", "number"),
-            WrongType("should-be-list", 80, "list", "string"))
```

Comment:
it isn't clear to me whether this should have been removed - was the test using an empty string? could we keep the test by changing it to a non-empty string? It seems like we'd still want to test this case for non-empty strings ?